c++/src/ColumnWriter.cc

```diff
@@ -840,15 +840,15 @@ namespace orc {
   /**
    * Implementation of increasing sorted string dictionary
    */
-  class StringDictionary {
+  class SortedStringDictionary {
```

Comment:
This renaming doesn't seem to relate to the purpose of this change. Suggest separate it.

Reply:
This change is to avoid the StringDictionary name conflicts cause we already declared StringDictionary in Vector.hh